deploy/20_cloud-ingress-operator.ClusterRole.yaml

```diff
@@ -20,4 +20,10 @@ rules:
   verbs:
   - patch
   - update
-  - watch
\ No newline at end of file
+  - watch
+- apiGroups:
+  - authorization.k8s.io
+  resources:
+  - subjectaccessreviews
+  verbs:
+  - create
```

Comment:
Note that I re-used the existing cloud-ingress-operator service account and role for the validating webhooks. It may be wise in the future to separate them to ensure both components follow least privilege principle.